### ISSUE TITLE
mdTabs: md-swipe-content doesn't work with 0.10.0-rc2

### ISSUE THREAD

**Issue by yellowiscool** [angular/material#3225]
I saw the new parameter md-swipe-content, but I can't get it working using the version 0.10.0-rc2.

| Version | CodePen | Works |
| ---------| ----- | ------- |
| 0.10.0-rc2 | http://codepen.io/anon/pen/QbgObY | ❌ |
| 0.9.8 | http://codepen.io/anon/pen/MwoOwN | ✓ |

**Comment by NetvisionBenjamin** [comment]
Same here

**Comment by eXaminator** [comment]
Seems to be still broken in 0.10.0.

**Comment by patriziotufarolo** [comment]
same here...

**Comment by tinocosta84** [comment]
same here, doesn't work

**Comment by nbraquart** [comment]
Same here with version 0.10.0.

**Comment by vniche** [comment]
+1
Mobile interfaces are very dependent of this feature.

**Comment by tinocosta84** [comment]
I am working on a solution guys, give me a day. #FingersCrossed

**Comment by tinocosta84** [comment]
for some reason, this function, although mentioned in material, doesn't actually exist, so let's create it!

**Comment by vniche** [comment]
lol, that's all you had to change to get it working? my hack was to force a click event on the next/previous tab with $timeout hahahaha shame on me

**Comment by giriprashaad** [comment]
my 2 cents in here, since its broken in the 0.10.0 version, i thought i'll comment in my workaround for others who are looking for a temporary fix.

I attached md-swipe-left="onSwipeLeft()" to the <md-content> inside a tab and md-selected="selectedIndex" for the <md-tabs>.

Added the following to work the swipes.
$scope.selectedIndex = 0;
$scope.onSwipeLeft = function(){
    if($scope.selectedIndex < 2) {
      $scope.selectedIndex  = $scope.selectedIndex + 1;
    }
  }

Seems to working fine for me.
